torch/cpu/__init__.py

```diff
@@ -1,6 +1,109 @@
+r"""
+This package implements abstractions found in ``torch.cuda``
```

Comment:
Note that this is missing a few more things from the list at https://github.com/pytorch/pytorch/blob/999abd56a7be3d73f0d2e9ab06356c7281a3307d/torch/utils/backend_registration.py#L31

But that can be done in a follow up for sure.

Reply:
yes, I plan to follow this up with Protocol'ization of this design so we can have tests for this.